app/models/debate_outcome.rb

```diff
@@ -2,6 +2,5 @@ class DebateOutcome < ActiveRecord::Base
   belongs_to :petition, touch: true
 
   validates :petition, :debated_on, presence: true
-  validates :petition_id, uniqueness: true
```

Comment:
There's a uniqueness constraint on ``debate_outcomes.petition_id`` so (race conditions with ``validates_uniqueness_of`` notwithstanding) isn't it still worth having this validation?